library/src/main/java/com/spendesk/grapes/extensions/ViewExtensions.kt

```diff
@@ -71,47 +71,49 @@ fun View.setDrawable(
 /**
  * Sets this view to [View.VISIBLE] state.
  */
-fun View.visible() {
+internal fun View.visible() {
     visibility = View.VISIBLE
 }
 
 /**
  * Sets this view to [View.INVISIBLE] state.
  */
-fun View.invisible() {
+internal fun View.invisible() {
     visibility = View.INVISIBLE
 }
 
 /**
  * Sets this view to [View.GONE] state.
  */
-fun View.gone() {
+internal fun View.gone() {
     visibility = View.GONE
 }
 
 /**
  * Sets this view to [View.VISIBLE] if the given [show] is true. Sets this view to [View.GONE] otherwise.
  */
-fun View.visibleOrGone(show: Boolean) {
+@Deprecated("Use attribute [androidx.core.view.isVisible] instead", ReplaceWith("isVisible", "androidx.core.view.isVisible"))
+internal fun View.visibleOrGone(show: Boolean) {
     if (show) visible() else gone()
 }
```

Comment:
I don't think we should deprecate this method, but change its implementation instead: 
```suggestion
internal fun View.visibleOrGone(show: Boolean) {
    isVisible = show
}
```


This is the purpose of this method, it abstracts the actual implementation. Also, it gives context to whatever happen when the view is not visible: is it gone or invisible? (Goes along with the method `visibleOrInvisible()`.)

Bonus: we don't need to update the code everywhere this method is called.